Makefile

```diff
@@ -6809,19 +6811,6 @@ endif
 
 
 LIBGOOGLE_BENCHMARK_SRC = \
```

Comment:
Removing these files seem wrong. Can you do a `git submodule update --init` (that will get the `third_party/google_benchmark/*` files to your machine) and then run  `generate_projects.sh` again?